README.md

```diff
@@ -154,7 +155,7 @@ serialize = "function used to serialize scalar"
 parse = "function used to create scalar instance from serialized form"
 ```
 
-All occurrences of `{graphql scalar name}` will be represented as `type`. If provided, `serialize` and `parse` will be used for serialization and deserialization.
+For each custom scalar there will generated annotation in `{scalars_module_name}.py`, it will be used in all occurrences of `{graphql scalar name}`. If provided, `serialize` and `parse` will be used for serialization and deserialization.
```

Comment:
```suggestion
For each custom scalar there will be an annotation generated in `{scalars_module_name}.py`, it will be used in all occurrences of `{graphql scalar name}`. If provided, `serialize` and `parse` will be used for serialization and deserialization.
```